VERSIONS.md

```diff
@@ -50,6 +50,10 @@ Example Labels
 }
 ```
 
+## Linux Package Versions
+
+To get a full listing of installed package versions, run: `docker run public.ecr.aws/aws-observability/aws-for-fluent-bit:3.0.0 grep "Installed:" /var/log/dnf.log /var/log/yum.log 2>/dev/null | sed 's/.*Installed: //'`. This can be useful to determine if a specific version is affected by a CVE.
+
 ## Version 3.0.0
```

Comment:
non blocking - Maybe we should be calling it Version 3.x here, as we bump up patch and build/OS versions